digdag-standards/src/main/java/io/digdag/standards/operator/HttpOperatorFactory.java

```diff
@@ -254,6 +254,15 @@ private ContentResponse execute(Request req, boolean uriIsSecret)
             catch (HttpResponseException e) {
                 throw error(req, uriIsSecret, e.getResponse());
             }
+            catch (RuntimeException e) {
+                logger.info("Exception without response: {} {}", req.getMethod(), safeUri);
+                if(retry){
```

Comment:
Could you put spaces before/after the parentheses?

Reply:
Yes. I fixed it.